lib/Importer/ONNXModelLoader.cpp

```diff
@@ -1050,6 +1050,41 @@ ONNXModelLoader::loadSpaceToDepth(const ONNX_NAMESPACE::NodeProto &op,
   return llvm::Error::success();
 }
 
+llvm::Error
+ONNXModelLoader::loadConstantOfShape(const ONNX_NAMESPACE::NodeProto &op,
+                                     const ArgumentDictionaryTy &dict) {
+  Tensor T(ElemKind::FloatTy, {1});
+  T.getHandle().raw(0) = 0.0;
+
+  if (dict.count("value")) {
+    RETURN_IF_ERR(loadTensor(dict.at("value")->t(), &T));
+    RETURN_ERR_IF_NOT(T.dims().size() == 1, "Value must be a 1D vector.");
```

Comment:
1D vector, makes sense, but doesn't it also need to be a single element? Perhaps this check should be `T.dims() == {1}`?